index.js

```diff
@@ -60,15 +60,53 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+function toggleColor() {
+  if (event.target.style.color){
+    if (event.target.style.color === "red") {
+      event.target.style.color = "black"
+    } else {
+      event.target.style.color = "red"
+    }
   } else {
-    element.style.color = "red"
+    event.target.style.color = "red"
   }
 }
 
+const h1 = document.querySelector("#header")
+h1.addEventListener("click", toggleColor)
+
 /***** Deliverable 2 *****/
 
+const playerClass = document.querySelector(".player")
+
+playerClass.addEventListener("click", function(){
+  const likeBtn = document.querySelector(".like-button")
+  const likeCount = document.querySelector("#like-count")
+
+  if (event.target === likeBtn) {
+    likeCount.innerText = parseInt(likeCount.innerText) + 1
+  }
+})
```

Comment:
Event delegation isn't really necessary here, since the only event we care about is the like button being clicked. Otherwise, looks good!

```js
const likeBtn = document.querySelector(".like-button")

likeBtn.addEventListener("click", function(){
  const likeCount = document.querySelector("#like-count")
  likeCount.innerText = parseInt(likeCount.innerText) + 1
})
```